packages/fast-components-react-base/src/toggle/examples.data.tsx

```diff
@@ -3,11 +3,28 @@ import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts
 import Toggle, { IToggleHandledProps, IToggleManagedClasses, IToggleUnhandledProps } from "./toggle";
 import * as schema from "./toggle.schema.json";
 import * as React from "react";
+import Documentation from "./.tmp/documentation";
 
 const examples: ISnapshotTestSuite<IToggleHandledProps & IToggleManagedClasses> = {
     name: "toggle",
     component: Toggle,
     schema: schema as any,
+    documentation: <Documentation />,
+    detailData: {
+        managedClasses: {
+            toggle: "toggle",
+            toggle_label: "toggle_label",
+            toggle_wrapper: "toggle_wrapper",
+            toggle_input: "toggle_input",
+            toggle_button: "toggle_button",
+        },
+        children: "Toggle",
+        id: "toggle01",
```

Comment:
Should id's be different in detailData vs data? You are using the same ids below

Reply:
This is elsewhere as well

Reply:
they don't need to be in this case since the views are never shown at the same time so the ids are always unique on the page

Reply:
cool